compiler/src/iree/compiler/Codegen/Common/MaterializeUserConfigs.cpp

```diff
@@ -194,6 +201,9 @@ struct MaterializeUserConfigsPass final
       //      ```
       LDBG("MaterializeUserConfigsPass on function: " << funcOp);
       if (succeeded(userTransformLibrary)) {
+        StringRef libraryModuleName =
+            userTransformLibrary->transformLibrary.getSymName().value_or(
+                "<unnamed>");
```

Comment:
nit: braces

Reply:
? the whole block is already in braces. I think you might have misread the indentation in this `if`